src/Symfony/Bridge/Doctrine/Form/DoctrineOrmTypeGuesser.php

```diff
@@ -179,4 +179,13 @@ protected function getMetadata($class)
             }
         }
     }
+
+    private static function getRealClass(string $class): string
```

Comment:
What about having a class in `doctrine/persistence` providing this, instead of making each package duplicate the doctrine naming convention (which would mean that changing it would be a BC break btw) ?

Reply:
I thought about that, I think we can possibly do that (probably immediately as `@deprecated` already since Proxy should be gone in ORM 3.0 / Persistence 2.0) and ship it as 1.1.